### PR TITLE
Show "unsupported" alert in Safari

### PR DESCRIPTION
**Pull request by bryanchriswhite**
This implements #36.

- [x] Add `UnsupportedModal` component
- [x] Use `UnsupportedModal` if we detect that we're in safari
- [x] Fix styling to match existing alert modal
- [x] Update copy

---

## Code Review Checklist (to be filled out by reviewer)

- [x] Description accurately reflects what changes are being made.
- [x] Description explains why the changes are being made (or references an issue containing one).
- [x] The PR appropriately sized.
~~- [ ] New code has enough tests.~~
~~- [ ] New code has enough documentation to answer "how do I use it?" and "what does it do?".~~
~~- [ ] Existing documentation is up-to-date, if impacted.~~

### CODE REVIEW

**Comment by bryanchriswhite** [comment]
Currently (1fc405e) it looks like this:
Screen Shot 2021-09-16 at 11 57 30 AM

**Comment by bryanchriswhite** [comment]
Currently (6e7e78d) it looks like this:
Screen Shot 2021-09-16 at 11 57 30 AM

**Comment by bryanchriswhite** [comment]
Sorry about the re-formatting.

File: src/worker.ts [hunk 1]
```diff
@@ -123,53 +129,55 @@ async function handleReceiveFileData({id}: RPCMessage): Promise<void> {
...
-    if (!isRPCMessage(event.data)) {
-        throw new Error(`unexpected event: ${JSON.stringify(event, null, '  ')}`);
-    }
-
-    // NB: unregister worker message handler.
-    //  (use message channel port instead)
-    onmessage = () => {
-        // NB: noop
-    };
+if (!browserIsProbablySafari) {
```
Comment (bryanchriswhite) [inline src/worker.ts:132]: @reviewer the diff here is adjusting indentation between (new) lines 133-182 and wrapping everything that would run on worker startup in the conditional expression on this line.

File: src/store/index.ts [hunk 2]
```diff
@@ -39,7 +41,23 @@ if (process.env['NODE_ENV'] === 'production') {
     host = 'https://wormhole.bryanchriswhite.com';
 }
 
-let client = new ClientWorker(defaultConfig);
+let client: ClientWorker;
+
+const safariNotSupportedError = Error("Safari is currently not supported but will be in future releases.")
```
Comment (on-ba) [inline src/store/index.ts:46]:
```suggestion
const safariNotSupportedError = Error("We plan to support Safari in future releases. Please try with a different browser.")
```

**Review by on-ba — APPROVED** [review]

**Pull request merged**